### PR TITLE
fix missing package and registering without a profile image

### PR DESCRIPTION
**Pull request by calhong**

### CODE REVIEW

File: src/secret.js [hunk 1]
```diff
@@ -0,0 +1 @@
+exports.SPOTIFY_CLIENT_SECRET = 'b73a36af4425400f8be932a05373436e'
```
Comment (vandyliu) [inline src/secret.js:1]: probably need to remove this

Comment (ianmah) [inline src/secret.js:1]: oopsies

File: package.json [hunk 2]
```diff
@@ -11,8 +11,9 @@
     "react-dom": "^17.0.2",
     "react-redux": "^7.2.4",
     "react-scripts": "4.0.3",
-    "recharts": "^2.0.9",
     "react-sliding-pane": "^7.0.0",
+    "react-spotify-auth": "^1.3.5",
```
Comment (vandyliu) [inline package.json:15]: we don't use this package anymore because it doesn't give refresh tokens

File: src/secret.js (hunk 1 above)
Comment (calhong) [reply src/secret.js:1]: OH MY GOD

File: package.json (hunk 2 above)
Comment (calhong) [reply package.json:15]: the package is a dependency inside one of the files, are we removing this now? It wouldn't build without this package

File: backend/server.js [hunk 3]
```diff
@@ -103,12 +103,13 @@ app.get('/callback', async (req, res) => {
     const user = await UserController.directFindUserBySpotifyId(data.body.id);
     if (!user) {
       console.log("No user found, creating new user");
+      let image = ((data.body.images[0].url != null) ? data.body.images[0].url : "");
```
Comment (vandyliu) [inline backend/server.js:106]: nit: 
```suggestion
      const image = ((data.body.images[0].url != null) ? data.body.images[0].url : "");
```

File: package.json (hunk 2 above)
Comment (vandyliu) [reply package.json:15]: good catch, 
I forgot to remove 
`import "react-spotify-auth/dist/index.css";`
in Login.jsx

You wanna remove it in this PR?

**Pull request closed without merging**